### ISSUE TITLE
english-lint --help

### ISSUE THREAD

**Issue by vi** [killercup/english-lint#4]
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/libcore/result.rs:1009:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

**Comment by killercup** [comment]
Yep, there's no nice CLI what-so-ever. Super embarrassing, seeing as just two years after writing this tool I actually ended up writing [a book](https://rust-lang-nursery.github.io/cli-wg/) on how to make good CLI apps in Rust!

I probably won't have time to work on this before Christmas, but if you or anyone else want to dig in: Add structopt and it's good to go! (Ideally as an optional dependency only used in the binary.)
